src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -633,45 +638,48 @@ static void window_cheats_money_mousedown(rct_window* w, rct_widgetindex widgetI
             widget_invalidate_by_class(WC_CHEATS, WIDX_MONEY_SPINNER);
             break;
         case WIDX_ADD_MONEY:
-            game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_ADDMONEY, _moneySpinnerValue, GAME_COMMAND_CHEAT, 0, 0);
-            break;
+        {
+            auto setCheatAction = SetCheatAction(CheatType::CHEAT_ADDMONEY, _moneySpinnerValue);
+            GameActions::Execute(&setCheatAction);
```

Comment:
This is used so many times just create a function that takes type of cheat and Param